ha/core/controllers/pcs/pcs_controller.py

```diff
@@ -218,12 +218,14 @@ def is_valid_node_id(self, node_id) -> bool:
                 raise HAClusterCLIError(f'{node_id} not a valid node_id: {err}')
         return True
 
-    def _is_node_in_cluster(self, node_id):
-        '''
-           Checks if node_id present in cluster or not
-           Returns: bool
-           Exception: HAInvalidNode
-        '''
+    def _is_node_in_cluster(self, node_id: str):
```

Comment:
is it validate node
or do we have diff for validate node

Reply:
It's same as previous validate_node() which checks whether node is present in cluster or not.